### ISSUE TITLE
Compliance to the Specs: Next Steps

### ISSUE THREAD

**Issue by DLu** [openelections/specs#8]
So one of my reasons for filing #6 and #7 was to formalize some rules, based on what I observed in the data, and then figure out where the data diverged from those rules. 

I've compiled my results into [this spreadsheet](https://docs.google.com/spreadsheets/d/1MXaL8XTBTSQQUe2-Eh52eKu_sDlIG9yUB83xf-loPX8/edit?usp=sharing). 

Now the question is what to do next. There are two primary options: 
 1. Expand the spec to cover more of the use cases found in the data
 2. Clean up the data to have fewer deviations from the spec

I'm opening this issue to start the discussion. To start, 
 * there's probably a decent cast for `winner` to be made an optional field
 * the Arkansas data should be cleaned up so that it uses `early_voting` instead of `early_vote`

**Comment by warwickmm** [comment]
> Now the question is what to do next. There are two primary options:
>
> 1. Expand the spec to cover more of the use cases found in the data
> 2. Clean up the data to have fewer deviations from the spec

My vote would be for option 2.  The more homogeneous the data, the easier it is to consume as well as test.

**Comment by DLu** [comment]
> > 2\.
> 
> My vote would be for option 2. The more homogeneous the data, the easier it is to consume as well as test.

Option 2 is great in some places, but I don't think we should remove useful data, as long as its use is documented.

**Comment by dwillis** [comment]
I agree: as few deviations as possible, but don't collapse distinct things into a single column. winner should be made an optional column; most states do not use it.
